crates/nu-cli/src/completion/command.rs

```diff
@@ -0,0 +1,126 @@
+use std::fs::{read_dir, DirEntry};
+use std::iter::FromIterator;
+
+#[cfg(all(windows, feature = "ichwh"))]
+use ichwh::{IchwhError, IchwhResult};
+use indexmap::set::IndexSet;
+
+use crate::completion::{Context, Suggestion};
+use crate::context;
+
+pub struct Completer;
+
+impl Completer {
+    pub fn complete(&self, ctx: &Context<'_>, partial: &str) -> Vec<Suggestion> {
+        let context: &context::Context = ctx.as_ref();
+        let mut commands: IndexSet<String> = IndexSet::from_iter(context.registry.names());
+
+        //let complete_from_path = config::config(Tag::unknown())
+        //    .map(|conf| {
+        //        conf.get("complete_from_path")
+        //            .map(|v| v.is_true())
+        //            .unwrap_or(true)
+        //    })
+        //    .unwrap_or(true);
+        let complete_from_path = true;
```

Comment:
I feel like we should always complete from the path.

If we disagree with that, and want to keep the old behaviour, I'll have to find a way to provide the configuration.